unlock-js/src/__tests__/web3Service.test.js

```diff
@@ -8,13 +8,13 @@ import Web3Service, { TransactionType, keyId } from '../web3Service'
 
 const nodeAccounts = [
   '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
-  '0xaca94ef8bd5ffee41947b4585a84bda5a3d3da6e'
+  '0xaca94ef8bd5ffee41947b4585a84bda5a3d3da6e',
```

Comment:
this seems odd to include, want to split it out?

Reply:
I think it might not pass CI without these comma additions

Reply:
definitely. Just was thinking it might be a good pre-PR, since if we need to revert this one, it would also nuke the commas, but I approved it because I don't feel strongly

Reply:
Ahhh I see. Yeah I'm amenable to either approach

Reply:
I just tried to spliut that up but of course I would have to add lint rules in the first PR that I am removing in the 2nd PR. So for this time I'll YOLO this!. Sorry :(